octobot_commons/logging/logging_util.py

```diff
@@ -139,6 +147,14 @@ class BotLogger:
     def __init__(self, logger_name):
         self.logger_name = logger_name
         self.logger = logging.getLogger(logger_name)
+        if SENTRY_ENABLED:
+            sentry_sdk.init(
```

Comment:
should we also do something on error/exception logs to push to sentry ?

Reply:
No it's automatically catch by sentry and uploaded

Reply:
wow nice !